### ISSUE TITLE
Web component: Leadspace - Graphic at the end of the main region is not announced.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#4315]
## Detailed description
On voiceover iPhone, JAWS FF

Issue: 
At the end of the main region, Screen reader is announcing a graphic as 
"image alt text " graphic

Expected: 
If this graphic conveys any meaningful information, convey the same using alt text. Else, hide the graphic from the screen reader using alt "".

**Comment by kennylam** [comment]
@Praveen-sr @RobertaJHahn The alt text "Image alt text" is for placement only. Meaningful text would be provided by the adopter. This is not a bug.
